lib/fluent/command/fluentd.rb

```diff
@@ -126,6 +126,18 @@
   opts[:without_source] = b
 }
 
+op.on('--config-file-type VALU', 'file type of fluentd. yaml or yml') { |s|
+  if (s == 'yaml') || (s == 'yml')
+    opts[:config_file_type] = s.to_sym
+  else
+    usage '--config-file-type accepts yaml or yml'
+  end
+}
+
+op.on('--guess-config-file-type', "Guess config file type that is yaml or not (default)", TrueClass) {|b|
```

Comment:
How about removing this and add `guess` to `--config-file-type` (and make it default).

Reply:
Sure, I'll change them.

Reply:
Fixed in https://github.com/fluent/fluentd/pull/3712/commits/49ec02315311c94ba3550c0b752ad24deaaaee0e

Reply:
:+1:

Reply:
Ah, sorry it's a bit different what I thought.
I intended adding `:guess` (`--config-file-type=guess`), how about this?

Reply:
Ah, I see. I'll try to implement what you thought.

Reply:
Really fixed in https://github.com/fluent/fluentd/pull/3712/commits/6574387ef2c78363dc23640df14d9886d5606bba